DQMOffline/Configuration/python/DQMOffline_cff.py

```diff
@@ -153,7 +154,9 @@
 
 #Post sequences are automatically placed in the EndPath by ConfigBuilder if PAT is run.
 #miniAOD DQM sequences need to access the filter results.
-PostDQMOfflineMiniAOD = cms.Sequence(miniAODDQMSequence*jetMETDQMOfflineSourceMiniAOD*tracksDQMMiniAOD*topPhysicsminiAOD)
+from DQMOffline.Muon.miniAOD_cff import * 
+
+PostDQMOfflineMiniAOD = cms.Sequence(miniAODDQMSequence*jetMETDQMOfflineSourceMiniAOD*tracksDQMMiniAOD*topPhysicsminiAOD*muonMonitors_miniAOD*MuonMiniAOD)
```

Comment:
@sscruz @dmitrijus - sorry for the slow review - but the reason for the delay turns out to be true (it seems) - MuonMinIAOD does not need trigger information, so it can live on a normal Path rather than an EndPath.